user-docker/run_competitor_container.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+CONTAINER="competitor-container"
```

Comment:
In relation with this, it is weird that we have a docker image with the postfix `-container`. It comes [from here](https://github.com/osrf/ariac-docker/pull/1/files#diff-5dc676f8954036c870eeb5f2425f757cR9). It should be easier if we name the docker images and containers something like:

Images: ariac-server (like now) and ariac-user
Containers: ariac-server-system (now is just test) and ariac-user-system

or any other combination with some consistence in naming.